common/supplemental/supplementalData.xml

```diff
@@ -3885,6 +3886,7 @@ XXX Code for transations where no currency is involved
 		</territory>
 		<territory type="SA" gdp="1775000000000" literacyPercent="87.2" population="33091100">	<!--Saudi Arabia-->
 			<languagePopulation type="ar" populationPercent="100" officialStatus="official"/>	<!--Arabic-->
+			<languagePopulation type="ars" populationPercent="2.99"/>	<!--Najdi Arabic-->
```

Comment:
If the language is normally not written (that's why I gave the example of Swiss German), then we set the writingPercent to low.

Reply:
Done